press/api/bench.py

```diff
@@ -563,10 +563,11 @@ def deploy(name, apps_to_ignore=[]):
 	if rg.deploy_in_progress:
 		frappe.throw("A deploy for this bench is already in progress")
 
-	candidate = rg.create_deploy_candidate(apps_to_ignore)
-	candidate.deploy_to_production()
+	candidates = rg.create_deploy_candidate(apps_to_ignore)
+	for candidate in candidates:
+		candidate.deploy_to_production()
 
-	return candidate.name
+	return candidates[0].name
```

Comment:
why not return all of them? This feels like a weird return type to me.

Reply:
This is only used for the frontend to go to the newer route. Ref: 

https://github.com/frappe/press/blob/cfd3285be1709974efe49c7a4bf0f4dc090703d3/dashboard/src/components/AlertBenchUpdate.vue#L112-L129

Reply:
Frontend can pick first?

Backend should ideall return all candidates that were created right?